gui/src/components/preboarding/login.vue

```diff
@@ -226,28 +124,17 @@ export default class CortxLogin extends Vue {
     username: "",
     password: ""
   };  
-  public resetAccountForm = {
-    password: "",
-    confirmPassword: ""
-  };
   public showResetPasswordDialog: boolean = false;
-  private passwordTooltipMessage = passwordTooltipMessage;
+  public ifResetPassword: boolean = true;
```

Comment:
This is not required now. You can remove it.

Reply:
removed variable